XLS-0033d-compact-fungible-tokens/README.md

```diff
@@ -171,33 +170,44 @@ A **`CFTokenIssuance`** can be removed using the same approach, but only if the
 Each **`CFTokenIssuance`** costs an incremental reserve to the owner account. This specification allows up to 32 **`CFTokenIssuance`** entries per account.
 
 #### 1.2.1.2. The **`CFToken`** object
-The **`CFToken`** object represents an amount of a token held by an account that is **not** the token issuer. CFTs are acquired via ordinary Payment or DEX transactions, and can optionally be redeemed or exchanged using these same types of transactions.
+The **`CFToken`** object represents an amount of a token held by an account that is **not** the token issuer. CFTs are acquired via ordinary Payment or DEX transactions, and can optionally be redeemed or exchanged using these same types of transactions. The object key of the `CFToken` is derived from hashing the space key, holder's address and the `CFTokenIssuanceID`.
```

Comment:
Let's break this last bit about the object ID into its own section called `CFTokenID`. We can make it look like [this section](https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0033d-compact-fungible-tokens#12111-cftokenissuance-ledger-identifier) (which is for CFTokenIssuanceID).

Reply:
Nevermind on the above comment. I'll submit a different PR to propose this change. Resolving this one.